Cargo.toml

```diff
@@ -8,16 +8,19 @@ clap = "^4.5"
 levenshtein = "1.0.5"
 rayon = "1.10.0"
 regex = "^1.10"
+mimalloc = { version = "^0.1", default-features = false }
 
 [dependencies.tes3]
 git = "https://github.com/Greatness7/tes3"
+branch = "dev"
 default-features = false
+# features = ["esp", "nightly", "simd"]
```

Comment:
Are these planned features?

Reply:
They are available only if you're on the rust's nightly toolchain (`rustup default nightly`). But they don't seem to give any significant perf improvement for this project.